### DISCUSSION TITLE
Can I use sidebar to navigate directly to a specific story in autodocs?

### DISCUSSION THREAD

**Discussion by KhaledSakr** [storybookjs/storybook#29362] [Help]
### Summary

I'm currently in the process of updating from storybook 6 to 8. In storybook 6, when you click on a specific story in the sidebar, it can navigate you directly to that story in the docs page (instead of the canvas).

Check this as an example: https://impargo.github.io/impargo-design-system/?path=/docs/components-form-form--disabled

However, in storybook 8, the sidebar entries only navigate to the canvas. And the docs page can only be accessed via the docs entry:
![image](https://github.com/user-attachments/assets/9477f57c-2d55-473b-aa83-1ea1608e8c65)

Is there a way to mimic the v6 behavior using the new autodocs?

### Create a reproduction

Can be reproduced using any default config from storybook.new. For example: https://stackblitz.com/github/storybookjs/sandboxes/tree/next/react-webpack/18-ts/after-storybook?file=.storybook%2Fpreview.ts

**Comment by shilman** [comment]
Please check out the table of contents option which is a kind of workaround 

https://storybook.js.org/docs/writing-docs/autodocs#generate-a-table-of-contents

**Reply by KhaledSakr** [reply]
Thanks for the suggestion! It really helps, but unfortunately it's a bit limited - also because of https://github.com/storybookjs/storybook/issues/29361
